### ISSUE TITLE
Load rows of system tables in parallel

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#4589]
Some of our system tables (e.g. `table_status`) perform operations that have to go over the network. They compute one row at a time, so if you have a slow network or one of the servers is busy, reading such a table can take a considerable amount of time.

We should probably parallelize this and compute multiple rows at once.

**Comment by danielmewes** [comment]
Did this for the `table_status` and `table_config` tables while working on some other things, because `table_status` was just too annoyingly slow. Will put up a code review soon.

**Comment by danielmewes** [comment]
Merged into next via 0744e95 and v2.3.x via 7ed1cf5.
